neuro_flow/batch_runner.py

```diff
@@ -70,7 +73,48 @@ async def run_subproc(self, exe: str, *args: str) -> None:
                 proc.kill()
                 await proc.wait()
 
-    async def bake(self, batch_name: str) -> None:
+    def _parse_action_name(self, action_name: str) -> LocalPath:
```

Comment:
These new `_collect_*` methods can also be methods of `BatchContext`, but I think it already does a lot, so I did not want to touch it in this PR.